CMakeLists.txt

```diff
@@ -225,7 +226,7 @@ set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "The integrated Rule-Oriented Data System"
 set(CPACK_DEB_COMPONENT_INSTALL OFF)
 set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS OFF)
 set(CPACK_DEBIAN_PACKAGE_CONTROL_STRICT_PERMISSION ON)
-set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA "${CMAKE_SOURCE_DIR}/packaging/postinst;")
+#set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA "${CMAKE_SOURCE_DIR}/packaging/postinst;")
```

Comment:
Remove

Reply:
Actually, I think this is probably a discussion point. `postinst` was giving me trouble when I moved the location of our configuration files out of `/etc/irods`, but I realized when I went to edit it that I didn't have an irods user or group on the machine where I was testing this stuff. Is this file really needed anymore? If not, I can remove this commented line as well as the file itself. If so, we need to discuss what we want it to do as far as permissions.

Reply:
eyeballing our main irods/irods postinstall.sh...

- clean up any /tmp files we might create/use

- handle any upgrades that might need to happen (probably not, since we will only ship template config files... we've learned that lesson)

- set up run levels

- ldconfig

- permissions...

Reply:
i think we're good to keep it out for now.   new issue coming soon to investigate this again in the future.

Reply:
Created #139 to address the issue of running the service as root which will necessarily include considerations for permissions on installed files, which is what the current `postinst` is doing.